LinqToDBPostGisNetTopologySuite.Tests/GeometryConstructorsTests.cs

```diff
@@ -12,13 +12,16 @@ namespace LinqToDBPostGisNetTopologySuite.Tests
     [TestFixture]
     class GeometryConstructorsTests : TestsBase
     {
+        private Version CurrentVersion { get; set; }
```

Comment:
I just added this field in most test classes in 9e429effb32e2190119c389b65d87011162163d2

Reply:
Is it better pulling CurrentVersion up as field of TestBase?

Reply:
Yes, getting current PostGIS version seems to be common code, may be moved to `TestsBase`

Reply:
In this commit or another?

Reply:
Would be better in another commit

Reply:
I will add them right now